scene/2d/tile_map.cpp

```diff
@@ -40,175 +40,291 @@
 #include "servers/navigation_server_3d.h"
 #endif // DEBUG_ENABLED
 
-Vector2i TileMapLayer::_coords_to_quadrant_coords(const Vector2i &p_coords) const {
-	int quad_size = get_effective_quadrant_size();
+#define TILE_MAP_DEBUG_QUADRANT_SIZE 16
```

Comment:
```suggestion
constexpr int TILE_MAP_DEBUG_QUADRANT_SIZE = 16;
```
`constexpr` is preferred over macros when defining constants (see canvas item editor plugin for example).
Also you can get rid of the weird assign below.